src/code/z_debug.c

```diff
@@ -267,7 +266,7 @@ void Regs_DrawEditor(GfxPrint* printer) {
     }
 }
 
-void func_80063D7C(GraphicsContext* gfxCtx) {
+void Debug_Draw(GraphicsContext* gfxCtx) {
```

Comment:
Despite this being in `z_debug.c` and called directly from `GameState_Draw`, `Debug_Draw` feels like a bit too generic of a name? It's primarily for `Debug_DrawOnScreenText` and `Regs_DrawEditor` right? Is `DebugText_Draw` or something else able to capture both of these? I'm not sure if you've had any other ideas. Might be fine as is though

Reply:
yea any more specific name I can think of makes it sound like its only for the printed screen text and not the reg editor. `DebugText_Draw` is an ok direction, but introducing a new system name for the one function is probably not ideal.
But at the same time, the two things it handles (camera screen text and reg editor) are kind of separate. so idk 

at the very least debug isnt wrong, it is two pretty seperate debugging features. but can understand feeling its generic.

I think in an ideal world other debug text in the game used this system and it would make more sense, but that is not how it played out in practice

Reply:
Yeah, the regs and text are different enough that it's hard to group them. Could make the other `Debug_` prefixes into `DebugText_` prefixes so it's not just one function, but then you'd have `Text` twice in those other functions. So `Debug_Draw` is probably fine, can't really think of a better alternative

Reply:
I think for now "this file is z_debug" is enough to name this as a "Debug" system.
We can change it later if needed, when more "debug only" stuff is named (not that there is a lot of it, is my impression), or if a better idea arises

In the meantime having this named at all is :+1: for me

Reply:
`Debug_DrawAllText` maybe?

Reply:
possibly. it does require that you know that this "system" only handles regs and ogawa's text lol
as opposed to the other systems which draw their own debug text with gfxprint

but I guess that context is needed no matter what we name this thing